src/styles/blocks/info.scss

```diff
@@ -0,0 +1,26 @@
+.info {
+  text-align: center;
+  width: 100%;
+  background-color: #fff;
+  padding: 10px;
+  border: 2px solid green;
+  border-radius: 15px;
+  visibility: hidden;
+
+  &__bold {
+    font-size: 20px;
+  }
+
+  &__text {
+    padding: 0;
+    font-size: 16px;
+
+    @media (max-width: 640px) {
+      font-size: 12px;
+    }
+
+    @media (max-height: 640px) {
+      font-size: 12px;
+    }
```

Comment:
Maybe combine this media queries?

Reply:
need to improve my knowledge in media queries, I try to make my work good in no mater what width or heigth, but as I recently mentioned there are some standarts in media queries, so it's not needed to make 10 media queries, it's enough to make like 320px mobile, 744px tablet, and 1024px pc or maybe 1440px or 1920px. need to work with it